src/System.Management.Automation/engine/CommandBase.cs

```diff
@@ -266,6 +266,23 @@ internal void InternalDispose(bool isDisposing)
 
 namespace System.Management.Automation
 {
+    #region ErrorView
+    /// <summary>
+    /// Defines the potential ErrorView options.
+    /// </summary>
+    public enum ErrorView
```

Comment:
If it is help enum and it is not in public contract/SDK maybe add obsolete attribute with warning?

Reply:
This will be part of public SDK